doc/en/user/source/data/database/postgis.rst

```diff
@@ -59,10 +86,63 @@ To begin, navigate to :menuselection:`Stores --> Add a new store --> PostGIS NG`
      - Time (in seconds) the connection pool will wait before timing out.
    * - :guilabel:`validate connections`
      - Checks the connection is alive before using it.
+   * - :guilabel:`Evictor run periodicity`
+     - Number of seconds between idle object evictor runs.
```

Comment:
This "evictor" stuff is probably magic to most users - they won't know what to do. We should provide some recommended values or at least general guidance.